lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -196,13 +196,11 @@ def get_ip(self, ue_id):
     def get_response(
         self,
         timeout: int = None,
-        assert_on_timeout: bool = True,
```

Comment:
This option was adding flexibility to this function that if required we could wait for extra time (more than default time) for specific messages. Is there any reason we want to remove this option?

Reply:
The option has not been used so far and has therefore been removed.

Reply:
@VinashakAnkitAman Are you okay with this option being removed since it's not currently used?

Reply:
Yah fine.. We can remove this option for now as per this change because it is not being used right now. Anyways we will be able to add it whenever needed again.